spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/Lars.scala

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.optim
+
+import com.intel.analytics.bigdl.Module
+import com.intel.analytics.bigdl.nn.Container
+import com.intel.analytics.bigdl.nn.abstractnn.{AbstractModule, Activity}
+import com.intel.analytics.bigdl.optim.SGD.{Default, LearningRateSchedule}
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.Table
+import org.apache.log4j.{Level, Logger}
+
+import scala.reflect.ClassTag
+
+
+/**
+ * An implementation of LARS https://arxiv.org/abs/1708.03888
+ * Lars.createOptimForModule is recommended to be used to create LARS optimizers for multiple layers
+ *
+ * @param lrScheduleOwner       if this optimizer owns the learning rate scheduler.
```

Comment:
change optimizer to optim method  to avoid confusion about what we call optimizer in BigDL